### PR TITLE
Rent Relief Act post 

### LINKED ISSUE #1988: Rent Relief Act post
https://blog.policyengine.org/how-the-rent-relief-act-of-2019-varies-with-income-and-location-f409a62e407c

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #1988

### CODE REVIEW

**Comment by PavelMakarchuk** [comment]
https://github.com/user-attachments/assets/c9ac1429-c395-4035-a52d-b96b7f7572ed

File: src/posts/posts.json [hunk 1]
```diff
@@ -799,5 +799,14 @@
     "authors": ["pavel-makarchuk"],
     "filename": "mn-hf1938-walz.md",
     "image": "mn-hf1938-walz.webp"
+  },
+  {
+    "title": "How the Rent Relief Act of 2019 Varies with Income and Location",
+    "description": "We show the impact on different households of the bill introduced by then-Senator Kamala Harris.",
+    "date": "2024-08-22 01:00:00",
+    "tags": ["us", "2024-election", "policy", "featured", "minnesota"],
```
Comment (MaxGhenis) [inline src/posts/posts.json:807]:
```suggestion
    "tags": ["us", "2024-election", "policy", "featured"],
```

File: src/posts/posts.json [hunk 2]
```diff
@@ -799,5 +799,14 @@
     "authors": ["pavel-makarchuk"],
     "filename": "mn-hf1938-walz.md",
     "image": "mn-hf1938-walz.webp"
+  },
+  {
+    "title": "How the Rent Relief Act of 2019 Varies with Income and Location",
+    "description": "We show the impact on different households of the bill introduced by then-Senator Kamala Harris.",
```
Comment (MaxGhenis) [inline src/posts/posts.json:805]:
```suggestion
    "description": "We show the impact on different households of the tax credit bill introduced by then-Senator Kamala Harris.",
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Use dashes in filenames (not underscores) and reduce the indentation on each heading

**Review by MaxGhenis — APPROVED** [review]

**Comment by PavelMakarchuk** [comment]
https://github.com/user-attachments/assets/97a9d5cf-0b81-4020-ae7f-9a1e580d10b0

**Pull request merged**